dependencies.yaml

```diff
@@ -348,10 +348,9 @@ dependencies:
               # TODO: Figure out what to do with this dependency
               # since the repo is now archived.
               - git+https://github.com/dask/dask-glm@main
-          # TODO: The conda-forge package is being generated from Leland's fork
-          # (https://github.com/lmcinnes/hdbscan) which is far behind now. Do
-          # we need to have that updated?
-          - hdbscan
+          # TODO: remove pin once a release that includes fixes for the error
+          # is released: https://github.com/rapidsai/cuml/issues/5514
+          - hdbscan<=0.8.30
```

Comment:
This won't fix the pinnings on pyproject, see just below here where it's pulling from git.

This pinning should move up, to the `- output_types: [conda, requirements, pyproject]` section. This package is now available on both conda-forge and PyPI, so we don't need a separate `conda` and `pyproject` pinning for it. Define this in the previous section and delete the git pinning below.

Reply:
If wheel CI is working fine and the only failures are in conda CI, please merge as-is and then make a second PR to unify the pinnings as I suggested above.

Getting CI unblocked before burndown is top priority.

Reply:
Yup, will address it in an immediate follow up PR